packages/utils/src/lib/reports/__snapshots__/generate-md-report.integration.test.ts.snap

```diff
@@ -421,7 +421,7 @@ ESLint rule **prefer-arrow-callback**. [📖 Docs](https://eslint.org/docs/lates
 
 ## About
 
-Report was created by [Code PushUp](https://github.com/flowup/quality-metrics-cli#readme) on Wed Sep 15 2021 00:00:00 GMT+0000 (Coordinated Universal Time).
+Report was created by [Code PushUp](https://github.com/flowup/quality-metrics-cli#readme) on Sep 15, 2021, 12:00 AM.
```

Comment:
suggestion: It's useful to include the time zone or UTC offset to avoid confusion and mental overhead of jugging time zone differences

Reply:
Good call, added it :+1: